src/content_script/pixel_inspector.js

```diff
@@ -146,6 +151,28 @@ glpPixelInspector.prototype.disable = function() {
     }
 }
 
+glpPixelInspector.prototype.bindFrameBuffer = function(enable) {
+  this.framebufferIsBound = !!enable;
```

Comment:
Turns it into a boolean value. So non-null values will be true instead of a WebGLBuffer